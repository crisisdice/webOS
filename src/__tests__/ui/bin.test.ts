import { runAll } from '../ui'
import { bin } from '../cases'

runAll(bin)
