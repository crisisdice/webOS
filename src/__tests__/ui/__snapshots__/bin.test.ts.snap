// Vitest Snapshot v1

exports[`cat "hello" > hello.txt<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ cat \\"hello\\" &gt; hello.txt</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`cd ../root<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ cd ../root</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">/home/root/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`cd /<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ cd /</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">//</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`cd ~<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ cd ~</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`echo "hello" > ../hello.txt<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ echo \\"hello\\" &gt; ../hello.txt</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`echo "hello" > hello.txt<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ echo \\"hello\\" &gt; hello.txt</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`echo hello<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ echo hello</span></div> <div class=\\"stdout svelte-1frjlh6\\">hello</div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`ls /<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ ls /</span></div> <div class=\\"stdout svelte-1frjlh6\\">home</div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`ls<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ ls</span></div> <div class=\\"stdout svelte-1frjlh6\\">.bashrc .history</div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`rm ../root<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ rm ../root</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`rm .bashrc<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ rm .bashrc</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`touch hello.txt<ENTER> 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ touch hello.txt</span></div> <div class=\\"stdout svelte-1frjlh6\\"></div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;
